app/assets/stylesheets/modules/home-page.scss

```diff
@@ -1,65 +1,91 @@
-.home-page-column h2 {
-  border-bottom: 1px solid $sul-h2-border-color;
-  color: $sul-h2-font-color;
+.article-home-page, .catalog-home-page {
+  h2, h4 {
+    font-weight: 400;
+    border-bottom: none;
+  }
 }
 
-.features {
-  .media {
-    img.media-object {
-      max-width: 95px;
-      @media only screen and (max-width: $screen-sm) {
-        max-width: 50px;
-      }
+.article-home-page {
+  .home-settings {
+    ul {
+        list-style: none;
+        margin-left: 0;
+        padding-left: 0;
     }
-  }
-}
 
-.home-page-column {
-  margin-top: 15px;
-  padding-left: 0;
+    .setting-label {
+      font-weight: bold;
+    }
 
-  &.articles-help {
-    padding-right: 0;
-  }
-  &.features {
-    @media only screen and (max-width: $screen-md) {
-      padding-right: 0;
+    .setting-tip {
+      margin-left: 17px;
+      margin-top: 5px;
     }
   }
-  @media only screen and (max-width: $screen-sm) {
-    padding-right: 0;
+
+  .home-page-tips {
+    .panel-heading {
+      color: $cardinal-red;
+    }
   }
 }
 
-.external-link-icon {
-  @extend .fa;
-  @extend .fa-external-link;
-  font-size: 0.9em;
-  margin-left: 2px;
+.catalog-home-page {
+  .features {
+    img {
+      height: 100px;
+      margin-right: 12px;
+    }
+  }
 }
 
-.media-heading {
-  font-size: 1.1em;
-}
+.home-page-bento {
+  .panel .panel-body {
+    padding: 0px;
+  }
 
-.article-home-page {
-  .home-page-column {
-    padding-right: 20px;
+  img {
+    width: 100%;
+    display: block;
+    margin: auto;
   }
-  ul {
-    list-style: none;
-    margin-left: 0;
-    margin-top: 0;
-    padding-left: 0;
-    padding-top: 0;
-    li {
-      font-size: .9em;
-    }
+
+  h4, p {
+    padding: 2px 10px;
   }
-  .features-body {
-    padding-bottom: 100px;
+
+  .home-page-you-are-here {
+    .panel .panel-body {
+      -webkit-box-shadow: 0px 0px 10px 0px rgba(233,131,0,1);
+      -moz-box-shadow: 0px 0px 10px 0px rgba(233,131,0,1);
+      box-shadow: 0px 0px 10px 0px rgba(233,131,0,1);
```

Comment:
I _think_ we can get away w/ just `box-shadow` here given that we have `autoprefixer-rails`.

Reply:
yeah i'm worried about changing this because i don't know how to test its removal 🤷‍♂️

Reply:
Yeah...actually, I just pulled those prefixes and checked the generated scss and they were not there as I expected.  :shipit: